modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/EditRelationship.tsx

```diff
@@ -73,7 +73,8 @@ export default function EditRelationship({
 		parameterRequired,
 	});
 
-	const readOnly = !hasUpdateObjectDefinitionPermission || values.reverse;
+	const readOnly =
+		!hasUpdateObjectDefinitionPermission || values.edge || values.reverse;
```

Comment:
Hi @murilostodolni,

the edge only should affect the deletion type SingleSelect component.

Reply:
Done

Reply:
Done